values.schema.json

```diff
@@ -47,11 +47,10 @@
       }
     },
     "keyValue": {
```

Comment:
It is not key value anymore, so name doesn't make sense, also this definition is not repeating so just inline it.